Add a line on contribute doc to ask author publishing Microsoft authorized template to azure samples browser (#404)
Azure/awesome-azd: website/docs/contribute.md

    - **Author's Name** - Name to credit on the gallery card
    - **Link to template source** - Link to the template GitHub repo
    - **Tags** - One or more [tags](https://github.com/Azure/awesome-azd/blob/main/website/src/data/tags.tsx) representing the template. Provide at least 1 tag for programming language used and at least 1 tag for Azure services integrated. Also tag the IaC provider (Bicep or Terraform). If you don't see a relevant tag for your template? Feel free to add one!

> 2. If the template is microsoft authorized, we encourage to [publish the template to azure samples browsers](https://review.learn.microsoft.com/en-us/help/contribute/samples/process/onboarding?branch=main). 

Reviewer: ```suggestion
2. If the template is Microsoft authorized, we encourage to [publish the template to Azure samples browsers](https://review.learn.microsoft.com/en-us/help/contribute/samples/process/onboarding?branch=main). 
```

Reviewer: What does Microsoft authorized mean? Is this one that is authored by a Microsoft employee? Or part of a Microsoft curated collection of templates?

> 2. If the template is microsoft authorized, we encourage to [publish the template to azure samples browsers](https://review.learn.microsoft.com/en-us/help/contribute/samples/process/onboarding?branch=main). 

Reviewer 2: ```suggestion
2. If the template is Microsoft-authored, we encourage you to also [publish it to learn.microsoft.com/samples](https://review.learn.microsoft.com/en-us/help/contribute/samples/process/onboarding?branch=main). 
```

3. Open a PR!

> 4. If possible, add a link to the PR in your repo where you made your app `azd` compatible to the PR description. This will help us provide feedback on your template and speed up the review process. 

Reviewer: I like this.
